src/pages/guides/sellers/extension-version.md

```diff
@@ -32,9 +32,3 @@ For existing, approved extensions, follow these instructions to add new versions
 You can continue with technical and marketing submissions, providing the code package, complete information, documentation, and more.
 
 You can save progress and edit the extension information as needed.
```

Comment:
Above, step 5 "Select the Magento Compatibility for the extension including the Magento edition (Open Source or Commerce) and version." Update to something like "Select the Compatibility for the extension including the Adobe Commerce or Magento Open Source edition.